source/Item.cpp

```diff
@@ -0,0 +1,87 @@
+#include"Item.hpp"
+
+namespace dungeon {
+
+Item::Item() {
+	hpAdd = atkAdd = 0;
+	hpPercentAdd = atkPercentAdd = 0;
+	critRateAdd = critDamageAdd = 0;
+}
+
+Item& Item::operator+=(const Item& item) {
+	hpAdd += item.hpAdd;
+	atkAdd += item.atkAdd;
+	hpPercentAdd += item.hpPercentAdd;
+	atkPercentAdd += item.atkPercentAdd;
+	critRateAdd += item.critRateAdd;
+	critDamageAdd += item.critDamageAdd;
+	return *this;
+}
+
+WreckingBar::WreckingBar() {
+	type = Class::any;
+	name = "Wrecking Bar";
+	hpAdd = 0;
+	atkAdd = 100;
```

Comment:
Wouldn't it be better to create a full constructor in Item and simply initialized it here rather than manually setting the value again in each child class?